src/fragments/lib/logging/common/setup_logging/setup_logging.mdx

```diff
@@ -0,0 +1,102 @@
+The Amplify Logger allows you to send existing log messages in the Amplify library to a AWS CloudWath log group that you provision and maintain.  You can also use the Amplify Logger to log your custom log messages and have it sent to the same Amazon CloudWatch log group.  Having the capability to access logs from your mobile application remotely via Amazon CloudWatch can help troubleshoot and debug your customer issues.
```

Comment:
The Amplify Logger enables you to troubleshoot and debug issues with your apps, to help you provide the best experience for your customers. You can log messages for errors by the Amplify library and add custom logs as well and send them to Amazon CloudWatch. With the Amplify Logger, you also can remotely change your logging configuration to adjust your logging levels, or add an allow list of customer IDs to help you detect issues more granularly for your apps in production.

Reply:
Text updated.